index.js

```diff
@@ -106,10 +106,10 @@ function plugin(opts) {
 */
 function getImages(css, opts) {
 	var images        = [];
-	var styleFilePath = css.source.input.file;
 
 	// Find only background & background-image declarations.
 	css.walkRules(function(rule) {
+		var styleFilePath = rule.source.input.file;
```

Comment:
This PR, however true it may be, is a breaking change from v2.0.1. In fact, it destroys all my sprite generation. I get no images. Please patch this in v2.0.3 by moving this line back above, where it was before. I manually made this change on a 2.0.2 install and it fixes everything.

This change can be released in v3.0 if you want, but it's totally breaking stuff.